packages/vscode-extension/src/telemetry/extTelemetry.ts

```diff
@@ -85,8 +85,10 @@ export namespace ExtTelemetry {
     const isExistingUser = getIsExistingUser();
     properties[TelemetryProperty.IsExistingUser] = isExistingUser ? isExistingUser : "";
 
-    const isSPFx = isSPFxProject(ext.workspaceUri.fsPath);
-    properties[TelemetryProperty.IsSpfx] = isSPFx.toString();
+    if (ext.workspaceUri) {
```

Comment:
ext.workspaceUri can be undefined when no workspace folder opened. Don't attach the property in that case.